torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -20,13 +27,141 @@ class TritonKernelWrapperFunctional(HigherOrderOperator):
     def __init__(self):
         super().__init__("triton_kernel_wrapper_functional")
 
-    def __call__(self, *, kernel, grid, kwargs):
-        kwargs = {
+
+triton_kernel_wrapper_functional = TritonKernelWrapperFunctional()
+
+
+@triton_kernel_wrapper_mutation.py_impl(DispatchKey.CompositeExplicitAutograd)
+def triton_kernel_wrapper_mutation_dense(*, kernel, grid, kwargs):
+    kernel[grid](**kwargs)
+
+
+@triton_kernel_wrapper_mutation.py_impl(FakeTensorMode)
+def triton_kernel_wrapper_mutation_fake_tensor_mode(mode, *, kernel, grid, kwargs):
+    with mode:
+        return None
+
+
+def trace_triton_kernel_wrapper(proxy_mode, func_overload, *, kernel, grid, kwargs):
+    with disable_proxy_modes_tracing():
+        out = func_overload(kernel=kernel, grid=grid, kwargs=kwargs)
```

Comment:
I believe this is always supposed to return `None`, right? Since this will redispatch to the `FakeTensorMode` implementation, which returns None (since this higher order op is only ever used for triton kernels).

Maybe add an assert?

Reply:
Only the mutation version returns none, but the functional version returns kwargs